content/references/http-websocket-apis/public-api-methods/clio-methods/server_info-clio.md

```diff
@@ -583,18 +583,23 @@ The `info` object may have some arrangement of the following fields:
 | `subscriptions.account`             |                 |   |
 | `subscriptions.accounts_proposed`   |                 |   |
 | `subscriptions.books`               |                 |   |
-| `load_factor`                       | Number          | The load-scaled open ledger transaction cost the server is currently enforcing, as a multiplier on the base transaction cost. For example, at `1000` load factor and a reference transaction cost of 10 drops of XRP, the load-scaled transaction cost is 10,000 drops (0.01 XRP). The load factor is determined by the highest of the [individual server's load factor](transaction-cost.html#local-load-cost), the cluster's load factor, the [open ledger cost](transaction-cost.html#open-ledger-cost) and the overall network's load factor. [Updated in: rippled 0.33.0][] |
+| `time`                              | String          | The current time in UTC, according to the server's clock. [New in: Clio v2.0](https://github.com/XRPLF/clio/releases/tag/2.0 "BADGE_BLUE") |
+| `uptime`                            | Number          | Number of consecutive seconds that the server has been operational. [New in: Clio v2.0](https://github.com/XRPLF/clio/releases/tag/2.0 "BADGE_BLUE") |
+| `amendment_blocked`                 | Boolean         | Whether the Clio server is [Amendment Blocked](amendments.html#amendment-blocked-in-clio-servers) [New in: Clio v2.0](https://github.com/XRPLF/clio/releases/tag/2.0 "BADGE_BLUE") |
```

Comment:
Is this field always provided when the value is `false`, or is it only part of the response if the server is amendment blocked? If it's optional, please add `_(May be omitted)_` to the start of the description.